### PR TITLE
fix typos in precompiles/EcPairing.yul

### PR DESCRIPTION
**Pull request by xiaolou86**
fix typos in precompiles/EcPairing.yul

### CODE REVIEW

File: precompiles/EcPairing.yul [hunk 1]
```diff
@@ -350,9 +350,9 @@ object "EcPairing" {
 			/// @dev Both input and output coordinates are encoded in Montgomery form.
             /// @dev If x and y differ from 0, just add z = (1,0).
             /// @dev If x and y are equal to 0, then P is the infinity point, and z = (0,0).
-            /// @param xp0, xp1 The x coordinate to trasnform.
+            /// @param xp0, xp1 The x coordinate to transform.
             /// @param yp0, yp1 The y coordinate to transform.
-            /// @return xr0, xr1, yr0, yr1, zr0, zr1 The projectives coordinates of the given G2 point.
+            /// @return xr0, xr1, yr0, yr1, zr0, zr1 The projectiles coordinates of the given G2 point.
```
Comment (ilitteri) [inline precompiles/EcPairing.yul:355]: This one was just a leftover "s"
```suggestion
            /// @return xr0, xr1, yr0, yr1, zr0, zr1 The projective coordinates of the given G2 point.
```

**Review by ilitteri — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just one minor detail and this PR can be merged.

File: precompiles/EcPairing.yul (hunk 1 above)
Comment (shuoer86) [reply precompiles/EcPairing.yul]: Thanks for your review. resolved.

**Comment by shuoer86** [comment]
@ilitteri Thanks for your review. I have resolved it.

**Comment by xiaolou86** [comment]
@ilitteri Thanks for your review. resolved.

**Pull request closed without merging**